spec/enumerables_spec.rb

```diff
@@ -0,0 +1,165 @@
+# frozen_string_literal: true
+
+require './lib/enumerable_number'
+
+RSpec.describe Enumerable do
+  describe 'my_each' do
+    let(:array) { [1, 2, 3, 4] }
+
+    it 'returns the original array' do
+      output = []
+      array.my_each { |element| output << element }
```

Comment:
all the specs should be wrapped within describe blocks, [reference](https://github.com/rubocop-hq/rspec-style-guide#empty-lines-around-examples).